di-ipv-core-stub/README.MD

```diff
@@ -0,0 +1,49 @@
+# Digital Identity IPV Core Stub
+
+`di-ipv-core-stub`
+
+## About
+
+An integration stub service for testing credential issuer journeys from a Stub IPV Core.
+
+This stub allows us to:
+
+* configure multiple credential issuers
+* set the user identity claim expected in the Experian UAT environment (name, date of birth, UKAddress)
+* set the user identity claim explicitly in a form (todo)
+* send these claims to a credential issuer
+* view the credentials returned
+
+## Environment
+
+Variable | Description | Example Value
+--- | --- | --- |
+CORE_STUB_PORT             | The port number the IPV Core Stub should run on | `8085` |
+CORE_STUB_CLIENT_ID              | The id of the IPV Core Stub client | `ipv-core-stub` |
+CORE_STUB_REDIRECT_URL               | The OAuth callback url | `http://localhost:8085/callback` |
+CORE_STUB_MAX_SEARCH_RESULTS   | Max search by name results | `200` |
+CORE_STUB_USER_DATA_PATH  | File path to Experian user data zip file | `/app/config/experian-uat-users-large.zip` |
+CORE_STUB_CONFIG_BASE64  | Base64 of the credential issuer config i.e. cris-dev.yaml ||
```

Comment:
It would be useful if in the future this was expanded to able to be a file path as well, and attached as a volume. That would allow for more easy use and switching of configuration.